### PR TITLE
Feature/ahc 1014 local ruby env

### PR DESCRIPTION
**Pull request by mkantzerFearless**
Confiogure local environment with docker?

### CODE REVIEW

**Review by solipet — CHANGES_REQUESTED** [review]

File: config/environments/docker-local.rb [hunk 1]
```diff
@@ -0,0 +1,107 @@
+Rails.application.configure do
```
[the remark this answers is not in the document]
Comment (mkantzerFearless) [reply config/environments/docker-local.rb]: I dont think this does mirror the development though, right?

File: config/database.yml [hunk 2]
```diff
@@ -21,3 +21,7 @@ test:
 production:
   pool: <%= ENV['DB_POOL'] || ENV['MAX_THREADS'] || 5 %>
   url: <%= ENV['DATABASE_URL'] %>
+
+docker-local:
+  <<: *default
+  url: <%= ENV['DATABASE_URL'] %>
```
[the remark this answers is not in the document]
Comment (mkantzerFearless) [reply config/database.yml]: I think that makes a lot of sense; dont set a default if it isnt used in multiple places / if everything is setting its own.

[the remark this answers is not in the document]
Comment (mkantzerFearless) [reply config/database.yml]: Although im a bit confused by dev/test taking `database` but prod taking a URL.

[the remark this answers is not in the document]
Comment (solipet) [reply config/database.yml]: We can set the component vars for all environments -- if you then set `DATABASE_URL` in your environment, that overrides the component vars.

File: config/environments/docker-local.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (solipet) [reply config/environments/docker-local.rb]: Erm, I _thought_ I did a diff against development and it was the same, but just tried again and ... nope. ¯\_(ツ)_/¯

nevermind... ;-)

**Comment by mkantzerFearless** [comment]
Dont Need

**Pull request closed without merging**